src/sentry/static/sentry/app/utils/consolidatedScopes.jsx

```diff
@@ -0,0 +1,137 @@
+import {pull, groupBy} from 'lodash';
+
+const PERMISSION_LEVELS = {
+  read: 0,
+  write: 1,
+  admin: 2,
+};
+
+const PERMISSION_VALUES = {
+  read: 'read',
+  write: 'read-write',
```

Comment:
why can't this be `write` and the label be `Read & Write` so that we don't need `PERMISSION_VALUES` ?